package.json

```diff
@@ -47,11 +47,12 @@
   },
   "dependencies": {
     "@azure/cosmos": "^3.6.3",
+    "@azure/identity": "^4.0.1",
```

Comment:
not sure if you already answered this somewhere else, but adding a note here from our offline conversation to consider making this a peer dependency and using type imports in the TS files where you use this